bin/mpdevil

```diff
@@ -843,15 +843,16 @@ class Client(MPDClient):
 	def _main_loop(self, *args):
 		try:
 			status=self.status()
+
+			self.emitter.emit("bitrate", int(status.get("bitrate", 0)))
```

Comment:
This will emit "bitrate" every 100 milliseconds even when the bitrate did not change. Why that choice?

Reply:
It's to avoid phantom values when switching from tracks that don't have valid bitrate info.
Going from a track with valid bitrate to one with invalid bitrate, the variable holding file type gets update but is never rendered since it's handled by` _on_bitrate()` leading it to display outdated info.

Reply:
That's another reason why I separated the top label. ;-) With the labels separated there is no need do defer the actual label update. But when we want to hide the bitrate when it's zero we get a strange "•" floating around. :-) What do you think of displaying a dash when the bitrate is really unknown? This would also be consistent with the duration labels.

Reply:
Oh sure, two dashes would look good too.
I merged the labels initially to avoid having to deal with the floating "•" (it could have been moved into _bitrate label but it starts to become a pile of hacks if more fields get added later).